pkg/collector/file.go

```diff
@@ -48,7 +48,8 @@ func (c *FileCollector) Get() ([]map[string]interface{}, error) {
 			input, err = ioutil.ReadFile(f)
 		}
 		if err != nil {
-			return nil, fmt.Errorf("failed to read file %s: %v", f, err)
+			log.Info().Msgf("failed to read file %s: %v", f, err)
+			continue
```

Comment:
If a file, that was explicitly passed via flag by the user, is not found, we should fail. I wouldn't change this - this is equivalent to failing to connect to the cluster in other collectors (or similar).